script/release

```diff
@@ -684,7 +684,7 @@ unless( $opts{T} // $release->config->skip_tests ) {
 if( $opts{t} // $release->config->dry_run ) {
 	$release->_print("============ Cleaning up\n");
 	$release->_print( "This is a dry run, so stopping. Cleaning up.\n" );
-	$release->distclean;
+	eval { $release->distclean };
```

Comment:
I wonder if I should change how `run` works so it doesn't croak. It can return success or failure instead. But I think this issue is unrelated to the issue, and we'd still need to ensure it outputs the error message.